advocacy_docs/edb-postgres-ai/analytics/external_tables.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Querying Delta tables in S3-compatible object storage
+navTitle: External Tables
+description: Access and Query data in S3-compatible object storage using External Tables
+deepToC: true
+---
+
+## Overview
+
+External tables allow you to access and query data stored in S3-compatible object storage using SQL. You can create an external table that references data in S3-compatible object storage and query the data using standard SQL commands.
+
+## Prerequisites
+
+An EDB Postgres AI account and a Lakehouse node.
```

Comment:
Can also use environment variables as per https://docs.google.com/document/d/1Td-57HLPngUAqLI3fz31n6khSg4EEqeUGY-nXHvqrYw/edit?tab=t.0#heading=h.2kmqh8ut3d4f

Reply:
Not on the cloud service instance; the user there does not have access to setting environment variables.

Reply:
But env vars are not a user accessible thing on CS.

Reply:
Yeah, I realised these are the docs for cloud, so envvars and filesystem-based storage locations don't apply.

There is a way to do credentialless auth because each cluster has an AWS identity that it picks up automatically and at some point I documented a multi-stage process that requires our AWS account admin and the user's account admin to both add some IAM policies allowing the cluster to read a bucket, but I don't think it's worth "officially" adding here.